### PR TITLE
Add compliance test for standard scs-0219-v1

### PR DESCRIPTION
**Pull request by kgube**

### CODE REVIEW

File: Tests/scs-compatible-kaas.yaml [hunk 1]
```diff
@@ -40,7 +53,8 @@ versions:
   - version: v1
     include:
       - ref: cncf-k8s-conformance
-      - ref: scs-0210-v2
-      - ref: scs-0214-v2
+      #~ - ref: scs-XXXX-v1
+      #~ - ref: scs-0210-v2
+      #~ - ref: scs-0214-v2
```
Comment (garloff) [inline Tests/scs-compatible-kaas.yaml:58]: This change looks unrelated to me.

**Comment by mbuechse** [comment]
Will be covered by #832.
